docs/src/pages/[platform]/theming/css-only.mdx

```diff
@@ -0,0 +1,67 @@
+import { Fragment } from '@/components/Fragment';
+import { Example, ExampleCode } from '@/components/Example';
+import { Authenticator } from '@aws-amplify/ui-react';
+import Link from 'next/link';
+import {
+  AuthenticatorButtonClassExample,
+  AuthenticatorClasses,
+} from './examples';
+
+A Theme is a structured collection of design decisions that change the appearance of a UI library. An Amplify UI theme allows application wide visual changes through the use of targeted CSS classes and variables.
+
+## Getting started
+
+<Fragment>
+  {({ platform }) =>
+    import(`../components/authenticator/quick-start-example.${platform}.mdx`)
+  }
+</Fragment>
+
+### Classes
+
+You can override the Authenticator's `amplify-*` classes to Theme the authenticator. For example to style all of the buttons in the Authenticator you could target the `amplify-button` class.
+
+<Example className="authenticator-theme-class-example">
+  <AuthenticatorButtonClassExample />
+
+  <ExampleCode>
+
+```css
+.amplify-button[data-variation='link']:hover {
+  background: none;
+  text-decoration: underline;
+}
+```
+
+  </ExampleCode>
+</Example>
+
+To view additional examples of styling the Authenticator using css classes please see our detailed <Link href={`/${props.framework}/components/authenticator#additional-css-styling`}>Authenticator</Link> documentation
+
+<AuthenticatorClasses />
+
+### Variables
+
+You can overwrite the Authenticator's `--amplify-*` css variables to theme the Authenticator. For example the sign in page of the Authenticator uses the `--amplify-components-fieldcontrol-*` css variables to style the sign in fields.
```

Comment:
Shouldn't this example say `--amplify-components-button-*` since that's the example below?
```suggestion
You can overwrite the Authenticator's `--amplify-*` css variables to theme the Authenticator. For example the sign in page of the Authenticator uses the `--amplify-components-button-*` css variables to style the sign in fields.
```